hello_nexus/README.md

```diff
@@ -0,0 +1,15 @@
+# Nexus
+
+Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, namespace,
+region, and cloud boundaries. It promotes a more modular architecture for sharing a subset of your team’s capabilities
+via well-defined service API contracts for other teams to use. These can abstract underlying Temporal primitives such as
+Workflows, or execute arbitrary code.
+
+Learn more at [temporal.io/nexus](https://temporal.io/nexus).
+
+The samples in this directory form an introduction to Nexus. 
+
+### Samples
+
+- [basic](./basic) - Nexus service definition, operation handlers, and calling workflows.
+- [without_service_definition](./without_service_definition) - A Nexus service implementation without a service definition 
```

Comment:
Not sure we want to show this in the samples, we should recommend using a service contract when calling across namespaces IMHO. The contract-less calls would have more value when we support same-team, direct task queue operation invocations.